projects/fourier/lib/fourier/commands/bundle.rb

```diff
@@ -40,21 +40,39 @@ def tuistenv
         required: false,
         aliases: :p,
       )
+      option(
```

Comment:
I think I need more context here. Seeing that we are exposing an argument to adjust the build directory makes me think that we are exposing APIs for debugging the build process. Is that what the argument is for?

Reply:
It is for debugging, but I believe it is fine since `bundle` command is for contributors only? It is very much useful if you need to debug the bundle command since a) build artifacts are in a directory you specify b) the build is reused across runs (not the case if is always run in a different temp dir)